### PR TITLE
Automatically move breakpoints into c64debugger

### PR DESCRIPTION
**Pull request by gitsoft**
### Automatically move breakpoints into c64debugger

**TLDR;**
KickAss exports breakpoints to a separate file so that VICE can use them. C64Debugger supports a breakpoint file, but it has another format then VICE. The code added in the pull request simply extract the breakpoints from the VICE file and create a separate file for C64Debugger when KickAss completed a build (_Build And Debug Startup_). 

**Nitty gritty + blah blah**

KickAss -vicesymbols output look like this. It contains labels and breakpoints.
>al C:837 .WaitForRasterLine
>al C:811 .mainloop
>al C:810 .start
>al C:80b .upstartEnd
>break 810 
>break 816 

To reuse this exported information I simply loop through the exported VICE symbol file and add any breakpoint line to a separate file. The result would look like this 
> break 810
> break 816 

Finally that new file (_with extension .breakpoints_) is passed with argument **-breakpoints** to the Slajerek's C64debugger.

Developed and tested successfully using
- Windows 10
- Kick assembler v5.0
- C64Debugger  v0.64.2
- vscode-kickass-c64 v1.0.0

**Verification in VS Code**
Add a breakpoint to a asm file using the usual format
> .break

Save.
Do a Build And Debug Startup
Open the .breakpoints in the binfolder to verify the entry exists

**Verification in C64Debugger**
After starting Build And Debug Startup, open the breakpoint window in C64Debugger using CMD + B (_CTRL + B on Windows_). The breakpoint will be listed there.

### CODE REVIEW

**Comment by CaptainJiNX** [comment]
Awesome Bernie. I think you forgot to install prettier though, but I'll merge this and reformat it.
